libc/src/__support/CPP/CMakeLists.txt

```diff
@@ -147,6 +147,8 @@ add_header_library(
     type_traits/true_type.h
     type_traits/type_identity.h
     type_traits/void_t.h
+  DEPENDS
+    libc.src.__support.macros.properties.float
```

Comment:
This is missing a bunch of deps, you might want to add them too (in a separate patch)

Reply:
Thx for noticing I'll fix this in a separate patch.